requirements-transitive.txt

```diff
@@ -79,7 +79,7 @@ sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.4
 SQLAlchemy==1.4.12
-thrift==0.14.1
+thrift @ git+https://github.com/apache/thrift.git@0.19.0#egg=thrift&subdirectory=lib/py
```

Comment:
If we cannot get the Thrift maintainers to upload a release to PyPI, we will need to use thrift-unofficial under our control to set up a release there, or rely on this specific github install directive.